### PR TITLE
Move sdk info to matter program folder

### PR DESCRIPTION
**Pull request by hiltonlima**
TH PR link: https://github.com/project-chip/certification-tool/pull/219

- Move SDK stuff from app/config.py to test_collections/matter/config.py
- Fixed exception when can't load a submodule on "app/test_engine/test_collection_discovery.py"
- Read SDK SHA only when matter program module is present
- Updated some scripts
- Move both update-paa-certs.sh and update-sample-apps.sh to matter program folder
-

### CODE REVIEW

File: scripts/pull-chip-tool-image.sh
Comment (hiltonlima) [inline scripts/pull-chip-tool-image.sh:1]: I didn't find where this script is called.

File: app/tests/version/test_read_test_harness_backend_version.py [hunk 1]
```diff
@@ -20,7 +20,8 @@
 import pytest
 
 from app import utils
-from app.core.config import settings
+# TODO: MATTER_SETTINGS
+from test_collections.matter.config import matter_settings
```
Comment (hiltonlima) [inline app/tests/version/test_read_test_harness_backend_version.py:24]: We can load the module dynamically and only test the SDK version if the matter module exists.
what do you think?

File: test_collections/matter/scripts/update-paa-certs.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+SDK_CERT_PATH="credentials/development/paa-root-certs"
+SDK_CERT_DEVELOPMENT_PATH="credentials/development"
+CERT_PATH="/var/paa-root-certs"
+DEVELOPMENT_PATH="/var/credentials/development"
+
+# If SDK path is not present, then do local checkout
+if [ -z "$SDK_PATH" ]
+then
+
+    # If SDK SHA is not present, then fetch from backend config
```
Comment (ccruzagralopes) [inline test_collections/matter/scripts/update-paa-certs.sh:42]:
```suggestion
    # If SDK SHA is not present, then fetch from config
```

Comment (hiltonlima) [reply test_collections/matter/scripts/update-paa-certs.sh]: Done

File: test_collections/matter/scripts/update-paa-certs.sh [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+set -x
+set -e
+
+
+# Usage: ./scripts/pi-setup/update-paa-certs.sh
```
[the remark this answers is not in the document]
Comment (hiltonlima) [reply test_collections/matter/scripts/update-paa-certs.sh]: Done

**Review by antonio-amjr — APPROVED** [review]

**Review by ccruzagralopes — APPROVED** [review]

**Pull request merged**